docs/user/aws/images/Makefile

```diff
@@ -0,0 +1,22 @@
+all: install_upi.svg install_upi_vpc.svg
+
+install_upi.svg: install_upi.dia AWS-Architecture-Icons_SVG/.touch AWS-Architecture-Icons_SVG/Light-BG/_Group\ Icons/Security-group_light-bg.svg
+	dia --export $@ $<
```

Comment:
do you think will be worth it to  add a note somewhere to say `dia` is a pre-req which can be installed using OS pkg tools - brew, apt, yum, dnf etc

Reply:
> do you think will be worth it to add a note somewhere to say `dia` is a pre-req which can be installed using OS pkg tools - brew, apt, yum, dnf etc

Won't folks without it get a useful "command not found" error already?  And I'd be surprised if all that many people were bumping the `.dia` source.  So I'm currently leaning towards "punt".  But I'd be fine landing docs if someone else wrote them up and they looked like they wouldn't go stale quickly ;).

Reply:
happy to add some notes (but will be in a week time once i'm back _online_)